modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml

```diff
@@ -0,0 +1,135 @@
+uuid: dbbff4a1-e9dc-4942-b91c-6533223d9055
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.media.az_large
+    - core.entity_view_mode.media.az_medium
+    - core.entity_view_mode.media.az_natural_size
+    - core.entity_view_mode.media.az_small
+    - core.entity_view_mode.media.az_square
+  module:
+    - bootstrap_utilities
+    - editor
+    - entity_embed
+    - ib_dam_wysiwyg
+    - media
+_core:
+  default_config_hash: Pf57ySytPg-TGLPWRmY40p-5YBwtdV07mz5mnKZYCoo
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.